tests/cli.test.ts

```diff
@@ -33,7 +34,7 @@ const tests: encodeTest[] = [
   ],
   ['mint(bool,bool,bool,bool)', ['true', 'false', '1', '0'], ['0x1', '0x0', '0x1', '0x0']],
   ['mint(bytes8)', ['0xffffffffffffffff'], ['0xffffffffffffffff']],
-  ['byte_me(bytes)', [['0xff', '0xff', '0xff', '0xff']], ['0x4', '0xff', '0xff', '0xff', '0xff']],
+  ['byte_me(bytes[])', [['0xff', '0xff', '0xff', '0xff']], ['0x4', '0xff', '0xff', '0xff', '0xff']],
```

Comment:
why did this behaviour change?

Reply:
The behaviour of `bytes` is similar to `string`: its an array of `byte`, but when you are going to parse a value hexadecimal you write it as a complete hexadecimal string, not as an array of hexadecimal values that can be parsed into one byte. 
An example of that is the previous function that receives `bytes8`. In this case, it is also an array of `byte`, but of fixed-size(8). The value you pass to it is `'0xffffffffffffffff'`, not `['0xff', '0xff', '0xff', '0xff']`.
Even if we would want to encode it that way, it may be confusing at the time of parsing. It would be similar to pass `string` values as explicit arrays of characters, like `['s', 't', 'r', 'i', 'n', 'g']`.